.github/workflows/verify.yml

```diff
@@ -370,6 +370,13 @@ jobs:
     - name: Execute 'xnvme library-info'
       run: xnvme library-info
 
+    - name: Build and install Python bindings
+      run: |
+        pip3 install pytest
+        cd toolbox/py_xnvme_pkg
+        make
+        make test
+
```

Comment:
Just a heads-up. You've only added it to `build-macos`. Linux is at line 326.

Reply:
Right. Fixed!
Windows and FreeBSD are still untested.
I have made a note of that in the TODO/Known Issues in ``docs/python/index.rst``.